src/main/java/mate/academy/rickandmorty/controller/CharactersController.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Сharacters management", description = "Endpoints for managing Сharacters")
+@RestController
+@RequestMapping("/characters")
+@RequiredArgsConstructor
+public class CharactersController {
+    private final CharacterService characterService;
+
+    @Operation(summary = "Get one random character")
+    @GetMapping("/randomly")
+    public CharacterEntity randomCharacter() {
+        return characterService.findRandom();
+    }
+
+    @Operation(summary = "Search characters by name")
+    @GetMapping("/search")
+    public List<CharacterEntity> searchByName(String name) {
```

Comment:
```suggestion
    @GetMapping
    public List<CharacterEntity> searchByName(String name) {
```
make name a request parameter